dev-tools/mage/pkg.go

```diff
@@ -142,6 +144,33 @@ func (b packageBuilder) Build() error {
 		b.Spec.Name, b.Type, b.Platform.Name)
 }
 
+// PackageSystemTests packages the python system tests results
+func PackageSystemTests() error {
```

Comment:
Umm `dev-tools/mage/pkg.go` contains targets and helpers to package builds of beats. I'd say this new target is more related to tests, consider moving it to other file.

You can also add it to to some directory under `dev-tools/mage/target` so it can also be imported in magefiles.

Reply:
Ok, I'll move this helper method to another file. Thanks for the review!